include/mbedtls/bignum.h

```diff
@@ -124,18 +124,42 @@
         #endif /* !MBEDTLS_HAVE_INT64 */
         typedef  int64_t mbedtls_mpi_sint;
         typedef uint64_t mbedtls_mpi_uint;
+        #if !defined(MBEDTLS_MPI_UINT_BITS)
+            #define MBEDTLS_MPI_UINT_BITS 64
+        #endif /* MBEDTLS_MPI_UINT_BITS */
+        #if !defined(MBEDTLS_MPI_HALF_MASK)
+            #define MBEDTLS_MPI_HALF_MASK 0xffffffff
+        #endif /* MBEDTLS_MPI_HALF_MASK */
+        #if !defined(MBEDTLS_MPI_HALF_BITS)
+            #define MBEDTLS_MPI_HALF_BITS 32
+        #endif /* MBEDTLS_MPI_HALF_BITS */
+        #if !defined(MBEDTLS_MPI_UINT_MASK)
+            #define MBEDTLS_MPI_UINT_MASK 0x7fffffffffffffffLL
+        #endif /* MBEDTLS_MPI_UINT_MASK */
     #elif defined(__GNUC__) && (                         \
         defined(__amd64__) || defined(__x86_64__)     || \
         defined(__ppc64__) || defined(__powerpc64__)  || \
         defined(__ia64__)  || defined(__alpha__)      || \
         ( defined(__sparc__) && defined(__arch64__) ) || \
         defined(__s390x__) || defined(__mips64)       || \
         defined(__aarch64__) )
-        #if !defined(MBEDTLS_HAVE_INT64)
+    #if !defined(MBEDTLS_HAVE_INT64)
```

Comment:
why is indentation change?

Reply:
My IDE did that. I'll correct it on my next commit.